ecosystem/typescript/sdk_v2/src/api/account.ts

```diff
@@ -140,4 +166,136 @@ export class Account {
     const resource = await getResource({ aptosConfig: this.config, ...args });
     return resource;
   }
+
+  /**
+   * Queries the count of tokens owned by an account
+   *
+   * @param accountAddress The account address
+   * @returns an object { count : number }
+   */
+  async getAccountTokensCount(args: { accountAddress: HexInput }): Promise<GetAccountTokensCountQueryResponse> {
+    const count = await getAccountTokensCount({ aptosConfig: this.config, ...args });
```

Comment:
yes.. this is because `await getAccountTokensCount` is imported from a file path `internal/account` and `async getAccountTokensCount` is a member class. if we were to do `await this.getAccountTokensCount` then we would end up with a loop =)